presentation/courses/addCourse.php

```diff
@@ -1,9 +1,10 @@
 <?php
+
```

Comment:
I think "addCourse" should probably be placed in a new folder (ie. "student"), since it refers to a student.

Reply:
It could, but it also refers to a course. Perhaps modify later as optional feature.